library/src/main/java/com/timehop/stickyheadersrecyclerview/HeaderPositionCalculator.java

```diff
@@ -19,6 +19,9 @@
   private final OrientationProvider mOrientationProvider;
   private final HeaderProvider mHeaderProvider;
   private final DimensionCalculator mDimensionCalculator;
+  
+  private final Rect bufferRect1 = new Rect();
+  private final Rect bufferRect2 = new Rect();
```

Comment:
Can we use more descriptive variable names? Also we should probably use the `m` prefix (although I now hate it) to be consistent